packages/yourloops/components/language-select.tsx

```diff
@@ -47,16 +47,17 @@ function LanguageSelect(): JSX.Element {
     if (Object.prototype.hasOwnProperty.call(i18n.options.resources, lang)) {
       const language = i18n.options.resources[lang].name;
       langs.push(
-        <MenuItem key={lang} value={lang}>
+        <MenuItem id={lang} key={lang} value={lang}>
           {language}
         </MenuItem>
       );
     }
   }
 
   return (
-    <FormControl>
+    <FormControl >

```

Comment:
```suggestion
    <FormControl>
```
Avoid a useless change will be helpful for the potentials merge conflicts in others PR.